Form/Type/MediaType.php

```diff
@@ -50,7 +50,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'new_on_update' => $options['new_on_update'],
         )));
 
-        $builder->addEventListener(FormEvents::BIND, function (FormEvent $event) {
+        $builder->addEventListener(FormEvents::SUBMIT, function(FormEvent $event) {
```

Comment:
`function (` is correct. A space must be there.

Please run `make cs` to have correct coding style.